### PR TITLE
build: Update 3rd party deps to latest versions

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?
Old versions of 3rd party images are used

## What is the new behavior?
Latest versions of 3rd party images are used

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/.env [hunk 1]
```diff
@@ -39,14 +39,15 @@ DEVICE_LLRP_VERSION=latest
...
+# Note that Vault images don't have none patch versions like 1.8
+VAULT_VERSION=1.8.0
+CONSUL_VERSION=1.10
 # Note that Postgres images don't use patch versions.
-POSTGRES_VERSION=12.3-alpine
-REDIS_VERSION=6.2.4-alpine
-KONG_VERSION=2.4.1-alpine
+POSTGRES_VERSION=13.3-alpine
+REDIS_VERSION=6.2-alpine
+KONG_VERSION=2.5
```
Comment (bnevis-i) [inline compose-builder/.env:48]: Verified that Kong never pushed the 2.5-alpine label and that it is just 2.5.

**Review by bnevis-i — DISMISSED** [review]

**Comment by lenny-intel** [comment]
recheck

**Review by jim-wang-intel — DISMISSED** [review]
LGTM

**Comment by lenny-intel** [comment]
@cherrycl , I am unable to determine the failure on this PR. Can you look at it please. THX!

File: docker-compose-arm64.yml [hunk 2]
```diff
@@ -446,7 +446,7 @@ services:
       STAGEGATE_SECRETSTORESETUP_TOKENS_READYPORT: '54322'
       STAGEGATE_WAITFOR_TIMEOUT: 60s
     hostname: edgex-kong
-    image: kong:2.4.1-alpine
+    image: kong:2.5
```
Comment (cherrycl) [inline docker-compose-arm64.yml:449]: @lenny-intel Got the error while pulling kong:2.5 in arm64 environment.
`INFO:EdgeX deployment:no matching manifest for linux/arm64/v8 in the manifest list entries`
Check on docker hub, the kong version is no linux/arm64/v8 image.

Comment (bnevis-i) [reply docker-compose-arm64.yml:449]: @hutchic Colin, was there a glitch in the Kong 2.5 release process?

File: compose-builder/.env (hunk 1 above)
Comment (lenny-intel) [reply compose-builder/.env]: Reverted back to 2.4

**Review by jim-wang-intel — APPROVED** [review]

File: docker-compose-arm64.yml (hunk 2 above)
Comment (hutchic) [reply docker-compose-arm64.yml]: Yes there was and it's since been remedied https://github.com/docker-library/official-images/pull/10737

Comment (lenny-intel) [reply docker-compose-arm64.yml]: @hutchic , Thank You!. Submit new [PR ](https://github.com/edgexfoundry/edgex-compose/pull/174) to upgrade to 2.5. Should pass TAF smoke tests this time with the ARM64 image available... ;-)

**Pull request merged**